src/Sentry.Unity.Editor/Native/BuildPostProcess.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using System.IO;
+using Sentry.Extensibility;
+using UnityEditor;
+using UnityEditor.Build;
+using UnityEditor.Callbacks;
+
+namespace Sentry.Unity.Editor.Native
+{
+    public static class BuildPostProcess
+    {
+        [PostProcessBuild(1)]
+        public static void OnPostProcessBuild(BuildTarget target, string pathToProject)
+        {
+            if (target is not (BuildTarget.StandaloneWindows or BuildTarget.StandaloneWindows64))
+            {
+                return;
+            }
+
+            var options = ScriptableSentryUnityOptions.LoadSentryUnityOptions(BuildPipeline.isBuildingPlayer);
+            var logger = options?.DiagnosticLogger ?? new UnityLogger(new SentryUnityOptions());
```

Comment:
Is `options` of type SentryUnityOptions? Can we instead do:

```suggestion
            var logger = options?.DiagnosticLogger ?? new UnityLogger(options);
```

Reply:
options may be null here. This is a line I've copy-pasted from another place (same pattern is used many times)